packages/teams-js/src/internal/utils.ts

```diff
@@ -37,6 +37,22 @@ function validateHostAgainstPattern(pattern: string, host: string): boolean {
   return false;
 }
 
+export function getDomainsFromCDN(): void {
+  fetch(validDomainsCdnEndpoint)
+    .then(async (response) => {
+      if (!response.ok) {
+        throw response.statusText;
```

Comment:
Does this error bubble up to the app and cause problems? I had assumed (maybe incorrectly) that if the CDN returns an error of some sort that we would use the fallback list and log that there had been an issue retrieving the CDN list.

Reply:
Tested, this doesn't cause issues for code execution, but you did catch something important which is to set `GlobalVariables.validOrigins` to `validOriginsFallback` in the case of the `fetch` call failing. I'm making the updates now. Great catch!